packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.d.ts

```diff
@@ -44,6 +44,10 @@ export interface TrapFocusProps {
    * @default false
    */
   disableRestoreFocus?: boolean;
+  /**
+   * Array of selectors to add to the components focusable elements
+   */
+  focusSelectors?: string[];
```

Comment:
It should be a function getter, to support APIs like https://github.com/focus-trap/tabbable.

Reply:
converted to a getter.  () => string[];